src/fe-text/terminfo-core.c

```diff
@@ -535,6 +537,11 @@ void terminfo_cont(TERM_REC *term)
 	if (term->TI_smkx)
 		tput(tparm(term->TI_smkx));
 
+	if (bracketed_paste_enabled) {
+		bracketed_paste_enabled = FALSE;
+		term_set_bracketed_paste_mode(TRUE);
```

Comment:
If `term_set_bracketed_paste_mode` didn't touch `term_set_bracketed_paste_mode` this could be just `term_set_bracketed_paste_mode(bracketed_paste_enabled)`, the code's too confusing